src/components/books.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import '../App.css';
+
+const Books = () => (
+  <div className="books">
+    <ul>
+      <li>
+        Beyond Order
+        <span>Jordan Peterson</span>
+      </li>
+      <li>
+        Mpas of Meaninng
+        <span>Jordan Peterson</span>
+      </li>
+      <li>
+        12 Rules for Life
+        <span>Jordan Peterson</span>
+      </li>
+      <li>
+        Sapiens
+        <span>Yuval Harrari</span>
+      </li>
```

Comment:
as required should have a placeholder button for removing books  🤓

Reply:
Done